RHEL/7/templates/static/ansible/disable_interactive_boot.yml

```diff
@@ -0,0 +1,14 @@
+# platform = multi_platform_rhel,multi_platform_fedora
```

Comment:
this needs to be in shared/templates/static/ansible and platform set to RHEL7 and multi_platform_fedora

This way it will never be picked up into Fedora benchmark.